docs/api/components/schemas.yaml

```diff
@@ -1835,7 +1835,81 @@ Database:
               items:
                 type: string
               example: ["CRUD", "HTTP", "Query"]
+        audit:
+          description: Audit logging configuration.
+          type: object
+          properties:
+            enabled:
+              description: Whether audit logging is enabled.
+              type: boolean
+              default: false
+            enabled_events:
+              description: List of enabled audit events
```

Comment:
Should this say that certain events will be enabled by default and have to be explicitly disabled?

Reply:
Had discussions with PM earlier this week. The plan is to have this represent the full set of enabled events. It contains the set of default events, and you can disable by removing from this list.